lib/bitcoin_burner.go

```diff
@@ -717,19 +718,19 @@ func CheckBitcoinDoubleSpend(txnHash *chainhash.Hash, blockCypherAPIKey string,
 }
 
 func BlockCypherPushAndWaitForTxn(txnHex string, txnHash *chainhash.Hash,
-	blockCypherAPIKey string, doubleSpendWaitSeconds float64, params *DeSoParams) error {
+	blockCypherAPIKey string, doubleSpendWaitSeconds float64, params *DeSoParams) (_err error, _isDoubleSpend bool) {
```

Comment:
Nit: Convention is generally to make the last argument the error. Would quick fix it here.